README.md

```diff
@@ -97,7 +97,7 @@ $ NIX_BUILD_GROUP_NAME=nixbuilder ./nix-installer install linux-multi --nix-buil
 
 ### Upgrading Nix
 
-You can upgrade Nix to the version described in [`DeterminateSystems/nix-upgrade`][nix-upgrade] by running:
+You can upgrade Nix to the current recommended version of Nix by running:
```

Comment:
Even though we changed how this upstream repo works, I'd still like to see a link to the file it has set as the latest.